test/handlers/admin/data-contributions.js

```diff
@@ -81,12 +81,19 @@ describe('admin data contributions handler', () => {
     });
 
     it('updates the DataContribution and redirects to /admin/data-contributions', () => {
-      const payload = {
-        approved: true,
-        curation_comments: 'My comments',
-      };
```

Comment:
No need to change it from a `const` to a `let`, as you can change constant dict's internal values. You can just initiate it with the hardcoded values (i.e. `approved`, `trial_id` and `curation_comments`) and then add the `data_category_id` when you create a new data category.